src/Autodesk.Forge.Core/ForgeHandler.cs

```diff
@@ -124,15 +132,24 @@ protected virtual IAsyncPolicy<HttpResponseMessage> GetResiliencyPolicies(int ti
                 },
                 onRetryAsync: (response, sleepTime, retryCount, content) => Task.CompletedTask);
 
-            // break circuit after 5 errors and keep it broken for 10 seconds
-            var breaker = errors.CircuitBreakerAsync(7, TimeSpan.FromSeconds(10));
+            // break circuit after 3 errors and keep it broken for 1 minute
+            var breaker = Policy
+                .Handle<HttpRequestException>()
+                .Or<Polly.Timeout.TimeoutRejectedException>()// thrown by Polly's TimeoutPolicy if the inner call times out
+                .OrResult<HttpResponseMessage>(response =>
+                {
+                    //we want to break the circuit if retriable erros persist or internal errors from the server
+                    return retriable.Contains((int)response.StatusCode) || 
+                           response.StatusCode == HttpStatusCode.InternalServerError;
+                })
+                .CircuitBreakerAsync(3, TimeSpan.FromMinutes(1));
```

Comment:
Does it mean we will `retry` after `1` min?

Reply:
No, this is circuit breaker. It means that we will not let any calls succeed for 1 minute if we receive 3 consecutive errors.